### ISSUE TITLE
Only visibility animations are supported

### ISSUE THREAD

**Issue by anarazel** [monperrus/ExpandAnimations#28]
Hi,

I tried to use this to export a slide deck with a few animations. It worked for some slides, but not others. Looking at the code the problem appears to be that only visibility changing animations are supported - the animations that didn't get split into separate pages were of type "Emphasis: Change Fill Color" (which I find useful for emphasis).

Regards,

Andres

**Comment by monperrus** [comment]
That's correct. Changed the issue title to "feature request: add support for "Emphasis: Change Fill Color""

Pull-request very welcome!
